ios/iosApp/UI/BaseViewModelViewController.swift

```diff
@@ -0,0 +1,29 @@
+import TrikotFrameworkName
+import UIKit
+
+class BaseViewModelViewController<V: BaseViewModelView<VM>, VMC: AbstractViewModelController, VM: ViewModel>: UIViewController {
+    var mainView: V {
+        view as! V
+    }
+
+    let viewModelController: VMC
+
+    init(viewModelController: VMC) {
+        self.viewModelController = viewModelController
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        view = V(frame: .zero)
+        mainView.viewViewModel = (viewModelController.viewModel as! VM)
+    }
```

Comment:
`loadViewI()` could be just this:
``` 
override func loadView() {
        view = V(viewModel: viewModel)
```

Reply:
Nope! This way we cannot unbind or models. We have specific case were we want to do the binding in the viewWillAppear and unbinding in viewDidDisappear.

Reply:
Good point too! I just haven't encountered the case in my projet, disregard my comment